### DISCUSSION TITLE
Celsius heat unit (IT) per second

### DISCUSSION THREAD

**Discussion by Joepieler** [JoelBender/bacpypes#427] [Ideas]
Is it possible to add Celsius heat unit per second to EngineeringUnits?

**Comment by JoelBender** [comment]
Yes, and there are two ways of accomplishing it.  The simplest is to extend the enumeration and add your own as a vendor value.  It is written:

> Enumerated values 0-255 and 47808-49999 are reserved for definition by ASHRAE. Enumerated values 256-47807 and 50000-65535 may be used by others subject to the procedures and constraints described in Clause 23.

The more time consuming one is to propose an update to the standard and submit it to the committee.  I'm sure there is an ASHRAE form to fill out someplace, but I can't seem to find it!

_Note:_ There are a few units in the 2020 version of the standard that have not been included in BACpypes, I will fix that. The addition of the split reserved range is irritating, but the nature of the beast.
